core/storage/feedback/gae_models.py

```diff
@@ -204,16 +299,34 @@ def get_messages(cls, exploration_id, thread_id):
 
     @classmethod
     def get_most_recent_message(cls, exploration_id, thread_id):
+        """Returns the most recent messages from the given thread, ordered
```

Comment:
Does this return all messages or just a set of "most recent" ones? If the latter, what determines whether a message is "most recent" or not?

(Please make sure you test this method and investigate thoroughly to ensure that it does what you think it does.)